browser/brave_rewards/checkout_dialog.cc

```diff
@@ -0,0 +1,298 @@
+/* Copyright 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_rewards/checkout_dialog.h"
```

Comment:
seems like this stuff should be under browser/ui/views/brave_rewards? @petemill ?

Reply:
I guess I'm not sure if it should be browser/ui/views/brave_rewards or just browser/ui/brave_rewards since it doesn't appear to have an ui/views dependencies

Reply:
@bridiver @petemill - `tip_dialog` also exists in the same directory - https://github.com/brave/brave-core/tree/master/browser/brave_rewards

Reply:
Yeah probably these should be in `browser/ui/brave_rewards` or `browser/ui/dialogs` (probably the former is nicer). Not `./views/` because they don't seem to involve the Views framework.

Reply:
Moved to `browser/ui/brave_rewards/`: https://github.com/brave/brave-core/pull/7557/files#diff-12af806eaa3f81d86bfdf05104c26ea18b11ae8e83a44517bb731466788902a0